tests/test_global_settings_handlers.cpp

```diff
@@ -267,6 +267,32 @@ TEST_F(TestGlobalSettingsHandlers, daemonRegistersHandlerThatAcceptsValidBackend
     ASSERT_NO_THROW(handler->set(key, val));
 }
 
+TEST_F(TestGlobalSettingsHandlers, daemonRegistersHandlerThatTransformsHyperVDriver)
+{
+    auto key = mp::driver_key, val = "hyper-v";
+    auto transformed_val = "hyperv";
+
+    mp::daemon::register_global_settings_handlers();
+
+    EXPECT_CALL(*mock_qsettings, setValue(Eq(key), Eq(transformed_val)));

```

Comment:
Another minor thing. 

Without specifying the called times, the default expected called number is at least once. Maybe we can make it more accurate by specifying times once since it is only called once. That makes

`EXPECT_CALL(*mock_qsettings, setValue(Eq(key), Eq(transformed_val))).Times(1)`